site/app/templates/AddTerm.twig

```diff
@@ -0,0 +1,45 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}add-new-term{% endblock %}
+{% block title %}Add New Term{% endblock %}
+{% block form %}
+    <form method="post" action="{{ add_term_url }}">
+        {{ parent() }}
+    </form>
+{% endblock %}
+{% block body %}
+    <input type="hidden" name="csrf_token" value="{{ csrf_token }}" />
+    <div class="flex-col my-3">
+        <div class="row">
+            <label class="col-md-6" for="term-id">
+                <div class="row">
+                    <span class="col-md-5">Term id:</span>
+                    <input type="text" name="term_id" placeholder="s20" id="term-id" class="col-md-6" required/>
+                </div>
+            </label>
+            <label class="col-md-6" for="start-date">
+                <div class="row">
+                    <span class="col-md-5">Start Date:</span>
+                    <input type="date" name="start_date" id="start-date" class="col-md-6" required/>
```

Comment:
this line fails the e2e tests for the following reasons.
> "The 'date' input type is not supported in all browsers. Please be sure to test, and consider using a polyfill."

it's more of a warning rather than an error, we can use this polyfill to fix this issue https://github.com/liorwohl/html5-simple-date-input-polyfill

Reply:
Or we can just ignore this warning as all of our targeted browsers support date and I refuse to have to cater to old dead browser versions.

Rewrite:

https://github.com/Submitty/Submitty/blob/44d39248f698f15d89f54b5c803705a0943f903f/tests/e2e/test_accessibility.py#L96-L101

to be:

```python
                for error in response.json()['messages']:
                    # For some reason the test fails to detect the doctype even though when you actually look at the rendered
                    # pages this error is not there. So therefore the test is set to just ignore this error.
                    skip_messages = [
                        "Start tag seen without seeding a doctype first",
                        "Possible misuse of “aria-label”",
                        "The 'date' input type is not supported in all browsers. Please be sure to test, and consider using a polyfill.",
                    ]
                    if error['message'] in skip_messages:
                        continue
```